src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "11.15.0",
+  "version": "11.16.0",
```

Comment:
there is no need to manually bump version -> it is done automatically via CI/CD

Reply:
@eoln - Fixed, please check now.